srcs/execution/builtins/echo.c

```diff
@@ -0,0 +1,139 @@
+#include "builtins.h"
+#include "opt.h"
+#include "libft.h"
+
+/*
+** for every arguments:
+** '-e': backslash escapes interpretation
+** handle '-s' and '-n' options
+*/
+
+int		builtin_echo(int argc, char **argv)
```

Comment:
I did not use use defines on the prompt as:
```bash
BUILTIN_RET builtin_echo(BUILTIN_ARGS)
```
Because it doesn't pass `norminette` with following message:
- `May not compile or is invalid for some reasons.`

What do we do?